src/sass/root.scss

```diff
@@ -0,0 +1 @@
+html, body, .app-root { height: 100% }
```

Comment:
can we add this class to the `formapp.scss` too?

Reply:
also I think html/body are covered elsewhere for the rest of the app.  only formapp has it's own css because it's based on bootstrap

Reply:
for that matter this can just be `.app-root` and we can import this scss over in formapp.js and remove the #root style from that stylesheet.

Reply:
Both are set by _base.scss, but that is only included by `provider-core` which isn't imported until the app starts up. So we also need to set them in this app-root file. Should I import this into `provider-core.scss` as well and remove the height: 100%; declarations from that one?

Reply:
ah yeah ok that makes some sense.. dynamic loading is so much fun.  Ok well maybe we pull it out of the `_base` then...  this means that it _doesn't_ need to be in `formapp.scss` at all then correct?

Reply:
Ok, I'll pull it out of `_base.scss` and confirm the formapp styles.